src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -71,6 +71,15 @@
     )
     public static boolean fixedAlexArms = true;
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Keyboard layout",
+        description = "The layout of your keyboard, used to fix input bugs accordingly.",
+        category = "Bug Fixes", subcategory = "Linux",
+        options = {"QWERTY", "AZERTY"}
+    )
+    public static int KeyboardLayout = 0;
```

Comment:
please use conventional java casing here, `KeyboardLayout` -> `keyboardLayout`

Reply:
Fixed, not sure why I did that